.github/actions/get-issue-url-and-version-number-v1/src/run.ts

```diff
@@ -0,0 +1,44 @@
+import type { Core, GitHub, Issue } from './types'
+import { getExecOutput } from '@actions/exec'
+
+export default async function run(core: Core, github: GitHub) {
+  const { repo, owner } = github.context.repo
+  try {
+    const {
+      stdout: rawIssue,
+      stderr: issueError,
+      exitCode: issueExitCode
+    } = await getExecOutput(
```

Comment:
This assumes that this code will always find an issue that matches. We should build in an extra guards for when that isn't the case